turbo.json

```diff
@@ -31,31 +38,47 @@
 				".custom-out-dir/types/**",
 				".svelte-kit/tsconfig.json"
 			],
-			"outputs": []
+			"outputs": [],
+			"outputMode": "new-only"
+		},
+		"format": {
+			"outputs": [
+				"src/**",
+				"scripts/**",
+				"shared/**",
+				"templates/**",
+				"types/**"
+			],
+			"outputMode": "new-only"
 		},
-		"format": {},
 		"prepare": {
 			"inputs": ["src/**", "source/**"],
 			"outputs": [
 				".svelte-kit/types/**",
 				".svelte-kit/tsconfig.json",
 				".custom-out-dir/types/**",
 				".custom-out-dir/tsconfig.json"
-			]
+			],
+			"outputMode": "new-only"
 		},
 		"test": {
 			"dependsOn": ["^build", "$CI", "$TURBO_CACHE_KEY"],
-			"outputs": []
+			"outputs": ["coverage/", "test-results/**"],
+			"outputMode": "new-only"
 		},
 		"lint": {
-			"outputs": []
+			"outputs": [],
+			"outputMode": "new-only"
 		},
 		"precommit": {
 			"dependsOn": ["format", "lint"],
-			"outputs": []
+			"outputs": [],
+			"outputMode": "new-only"
 		},
 		"dev": {
-			"cache": false
+			"cache": false,
+			"outputs": [".svelte-kit"],
```

Comment:
I think this is another one we don't need

Reply:
yeah, I think you're right since we have `"cache": false` here